views/templates/block/checklist.twig

```diff
@@ -35,7 +35,7 @@
                             {{ 'Your server is running on a supported PHP version.'|trans }}
                         {% else %}
                             {{ 'The PHP version your server is running on is obsolete and needs to be upgraded. [1]Learn more why[/1] or [2]ignore[/2].'|trans({
-                                '[1]': '<a href="' ~ phpUpgradeRequired|trans ~'" target="_blank">',
+                                '[1]': '<a target="_blank" href="' ~ phpUpgradeRequired|trans ~'" target="_blank">',
```

Comment:
Don't you see there is already a target="_blank"? :thinking: 
But adding `rel="noopener noreferrer nofollow"` could be cool :)